util/merkle/src/triedb.rs

```diff
@@ -105,6 +105,26 @@ impl<'db> TrieDB<'db> {
             None => Ok(None),
         }
     }
+
+    /// Check if every leaf of the trie exists
+    pub fn is_complete(&self) -> bool {
+        *self.root == BLAKE_NULL_RLP || self.is_complete_aux(self.root)
+    }
+
+    /// Check if every leaf of the trie starting from `hash` exists
+    pub fn is_complete_aux(&self, hash: &H256) -> bool {
```

Comment:
Should it be public?
If aux means auxiliary, I think it's not a right name for the public method.

Reply:
Good point. I think it shouldn't be public.
@foriequal0 Could you remove it?